WMFComponents/Sources/WMFComponents/Components/Shared/Feature Announcement/WMFFeatureAnnouncementView.swift

```diff
@@ -49,15 +48,27 @@ struct WMFFeatureAnnouncementView: View {
                                 .foregroundColor(Color(appEnvironment.theme.text))
                         }
                         if let image = viewModel.image {
-                            Image(uiImage: image)
-                                .resizable()
-                                .aspectRatio(contentMode: .fit)
-                                .frame(width: 132, height: 118)
-                                .foregroundColor(imageColor)
+                            ZStack {
+                                if let backgroundImage = viewModel.backgroundImage {
```

Comment:
@mazevedofs Not too sure what the cause is, but it'd be good if we can get this to shrink in width on iPad if needed:

![see](https://github.com/user-attachments/assets/2d64da3b-e385-4768-b85d-41d7631df1ca)

Reply:
I should be working on iPad and iPhone now 🙌